attestation/check-production.sh

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2024 Canonical Ltd.
+#
+# This file is part of tdx repo. See LICENSE file for license information.
```

Comment:
Should we write something more specific in the two sentences at the bottom? Or has this wording already been developed/approved by the powers at be? I'm thinking something like the following: "This file is part of Canonical's TDX GitHub repository (https://github.com/canonical/tdx). See the LICENSE file in the repository for license information."